client/components/main/Desktop/TypeSelector/index.js

```diff
@@ -56,65 +51,65 @@ const RequestTypeSelector = ({
   const checkAll = () => {
     toggle(!isToggled);
     return !isToggled;
-  }
+  };
 
-  const updateAll = (value) =>{
-      requestTypes.map(type => {
-        if(selectedTypes[type.typeId] !== value){
-          updateTypesFilter(type.typeId)
-        }
-      })
-  }
+  const updateAll = value => requestTypes.map(type => {
```

Comment:
Let's use `forEach` here since we don't need to create a new array

Reply:
sounds good, changes made